Update file for Windows compatibility

MaximeZmt opened the pull request:
This Pull Request contains the following modifications:

README.md:
After having lost a lot of time looking at how to compile the project on windows. 
We finally found a GCC 64 bits compiler that works well.
I added the URL into the documentation to avoid other Windows users losing time.

make.bat:
I've updated the build script to automatically rename pop into pop.exe. Like that, Windows will automatically be able to run it from the terminal.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/dedis/popstellar?pullRequest=857) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/dedis/popstellar?pullRequest=857) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/dedis/popstellar?pullRequest=857) it.</sub>

nkcr commented on be1-go/README.md:
```diff
@@ -84,6 +84,16 @@ sudo apt-get install build-essential # Ubuntu/Debian
 You may build the `pop` CLI to interact with the server by executing `make
 build` on Linux/OSX or `make.bat build` in Windows.
 
+> Special Note for Windows:
```
Would be better to have a subsection for each environment:

```
#### Unix

...

#### Windows

....
```

nkcr reviewed (commented):
Thanks! 👍

Warhedgehog commented:
Should we add that the user should make sure the environment variable are correctly set ?

MaximeZmt commented:
> Should we add that the user should make sure the environment variable are correctly set ?

I don't know. It is just that it is not done necessarily by default on windows (to set the path)! So It might be a case of failure. Then there is room for debate.

Warhedgehog commented:
You're probably right, making sure it is correctly set is part of installing it

Warhedgehog reviewed (approved):
LGTM 👍

MaximeZmt commented:
> You're probably right, making sure it is correctly set is part of installing it

Yet I've removed it following your comment. Should I add it back 😅 ? 

Otherwise, I've refactored a small part of the execution paragraph.

Warhedgehog commented:
> > You're probably right, making sure it is correctly set is part of installing it
> 
> Yet I've removed it following your comment. Should I add it back 😅 ?
> 
> Otherwise, I've refactored a small part of the execution paragraph.

Either way is fine with me, I'll leave you the judge of it. 
What just came to mind is that for the suggestion of the compiler we should add the date at which we consider it the best option. In the coming months or years, it might change quite a bit.

MaximeZmt commented:
> > > You're probably right, making sure it is correctly set is part of installing it
> > 
> > 
> > Yet I've removed it following your comment. Should I add it back 😅 ?
> > Otherwise, I've refactored a small part of the execution paragraph.
> 
> Either way is fine with me, I'll leave you the judge of it. What just came to mind is that for the suggestion of the compiler we should add the date at which we consider it the best option. In the coming months or years, it might change quite a bit.

I've added the date :)

Warhedgehog reviewed (approved):
Good job !
